mysql-test/suite/innodb/r/alter_copy_bulk.result

```diff
@@ -91,3 +91,23 @@ INSERT INTO t1 VALUES
 ALTER TABLE t1 FORCE, ALGORITHM=COPY;
 DROP TABLE t1;
 SET GLOBAL innodb_stats_persistent=@default_stats_persistent;
+#
+#  MDEV-36504 Memory leak after insert into empty table
+#
+CREATE TABLE t1 (k INT PRIMARY KEY)ENGINE=InnoDB;
+INSERT INTO t1 SET k= 1;
+START TRANSACTION;
+INSERT INTO t1 SET k= 2;
+SELECT COUNT(*) > 0 FROM mysql.innodb_index_stats LOCK IN SHARE MODE;
+COUNT(*) > 0
+1
+connect con1,localhost,root,,,;
+SET innodb_lock_wait_timeout=0;
+CREATE TABLE t2(f1 INT DEFAULT 1, PRIMARY KEY(f1))ENGINE=InnoDB as SELECT k FROM t1;
```

Comment:
This test appears to depend on `STATS_PERSISTENT=1` but it is not part of the table definition.

This test is using `CREATE TABLE…SELECT`. How does this behaviour depend on the current `innodb_alter_copy_bulk` setting? Is there any similar bug in `ALTER TABLE…ALGORITHM=COPY` when we are attempting to update the InnoDB persistent statistics?

Reply:
This issue shouldn't exist for alter table copy algorithm. CREATE..SELECT also depends on innodb_alter_copy_bulk variable. Because `row_ins_clust_index_entry_low()` does bulk insert based on `innodb_alter_copy_bulk` value.